### PR TITLE
chore/added migrations and test

### PR DESCRIPTION
**Pull request by gecBurton**
## Things to check

- [ ] I have added any new ENV vars in all deployed environments
- [ ] I have tested any code added or changed
- [ ] I have run integration tests

### CODE REVIEW

File: django_app/redbox_app/redbox_core/migrations/0056_alter_aisettings_retrieval_system_prompt_and_more.py [hunk 1]
```diff
@@ -15,6 +21,12 @@ class Migration(migrations.Migration):
             name='retrieval_system_prompt',
             field=models.TextField(default="You are a specialized GPT-4o agent. Your task is to answer user queries with reliable sources.\n**You must provide the sources where you use the information to answer.**\n- If the information is found in the provided documents, state that in the `text` field and provide citation details in the `citation` field.\n- If the information is not found in the provided documents, state that in the `text` field (e.g., 'The response to your question is not available in provided documents, however, this is what I found' and then provide your response.)\n\n{format_arg}"),
         ),
+        migrations.AlterField(
```
Comment (gecBurton) [inline django_app/redbox_app/redbox_core/migrations/0056_alter_aisettings_retrieval_system_prompt_and_more.py:24]: remove enums/choices so we can set value to be anything

File: django_app/redbox_app/redbox_core/migrations/0056_alter_aisettings_retrieval_system_prompt_and_more.py [hunk 2]
```diff
@@ -15,6 +21,12 @@ class Migration(migrations.Migration):
...
+            model_name='citation',
+            name='source',
+            field=models.CharField(default='UserUploadedDocument', help_text='source of citation', max_length=32),
+        ),
+        migrations.RunPython(change_citation_source, migrations.RunPython.noop),
```
Comment (gecBurton) [inline django_app/redbox_app/redbox_core/migrations/0056_alter_aisettings_retrieval_system_prompt_and_more.py:29]: set values to something legit

File: django_app/redbox_app/redbox_core/migrations/0056_alter_aisettings_retrieval_system_prompt_and_more.py [hunk 3]
```diff
@@ -15,6 +21,12 @@ class Migration(migrations.Migration):
...
         migrations.AlterField(
```
Comment (gecBurton) [inline django_app/redbox_app/redbox_core/migrations/0056_alter_aisettings_retrieval_system_prompt_and_more.py:30]: now apply new enums/choices

File: django_app/tests/test_migrations.py [hunk 4]
```diff
@@ -360,3 +360,41 @@ def test_0055_citation_source_citation_url_alter_citation_file(original_file, mi
...
+    chat_message = ChatMessage.objects.create(chat=chat)
+
+    File = old_state.apps.get_model("redbox_core", "File")
+    file = File.objects.create(
+        user=user,
+        original_file=original_file,
+        original_file_name=original_file.name,
+    )
+
+    Citation = old_state.apps.get_model("redbox_core", "Citation")
```
Comment (gecBurton) [inline django_app/tests/test_migrations.py:389]: create old Citation with source="USER UPLOADED DOCUMENT"

**Review by jamesrichards4 — APPROVED** [review]

**Pull request merged**